lib/core/package.json

```diff
@@ -123,6 +123,7 @@
     "url-loader": "^4.0.0",
     "util-deprecate": "^1.0.2",
     "webpack": "^4.43.0",
+    "react-docgen-typescript-plugin": "^0.5.1",
```

Comment:
why? this is react-specific

Reply:
It's imported here:
https://github.com/storybookjs/storybook/blob/4286a939f3a0d25336ec8182116a9c6dce1e10bf/lib/core/types/index.ts#L2

if it's imported, it should be specified as a dependency, no?

Reply:
@mrmckeb @hipstersmoothie any ideas how to avoid this?

Reply:
if the type appears in your output .d.ts files then you need have it as a dep